README.md

```diff
@@ -132,7 +132,8 @@ To set options and keep lint-staged extensible, advanced format can be used. Thi
 
 ## Filtering files
 
-It is possible to run linters for certain paths only by using [minimatch](https://github.com/isaacs/minimatch) patterns. The paths used for filtering via minimatch are relative to the directory that contains the `.git` directory. The paths passed to the linters are absolute to avoid confusion in case they're executed with a different working directory, as would be the case when using the `gitDir` option.
+It is possible to run linters for certain paths only by using [minimatch](https://github.com/isaacs/minimatch) patterns.
+The file patterns should be relative to the project root(where `lint-staged` is installed).
```

Comment:
> the project root(where `lint-staged` is installed)

this sound confusing to me. I'd say, project root is where `.git` is located and `lint-staged` is installed where `package.json` is located. Am I missing something here?

Reply:
Ya the terminology is a bit muddy here. Is this okay:

> The file patterns should be specified relative to the package root (where `lint-staged` is installed).